src/jsMain/js/simulator/windows/ShaderEditorWindow/ShaderEditorWindow.jsx

```diff
@@ -19,6 +20,8 @@ const ShowEditorWindow = (props) => {
   const statusContainerEl = useRef(null);
   const [glslPreviewer, setGlslPreviewer] = useState(null);
   const [gadgets, setGadgets] = useState([]);
+  const [openShaders, setOpenShaders] = useState([]);
+  const [menuEl, setMenuEl] = React.useState(null);
```

Comment:
Absolute nitpick comment here:

We already import `useState` at the top of this file, so we could do:
```
const [menuEl, setMenuEl] = useState(null);
```
instead of:
```
const [menuEl, setMenuEl] = React.useState(null);
```